attributes/repo.rb

```diff
@@ -18,6 +18,6 @@
 #
 
 # URL to repository
-default['cdap']['repo']['apt_repo_url'] = 'http://repository.cask.co/ubuntu/precise/amd64/releases'
-default['cdap']['repo']['apt_components'] = ['releases']
-default['cdap']['repo']['yum_repo_url'] = 'http://repository.cask.co/centos/6/x86_64/releases'
+default['cdap']['repo']['apt_repo_url'] = 'http://repository.cask.co/ubuntu/precise/amd64/cdap'
+default['cdap']['repo']['apt_components'] = 'node['cdap']['version'].to_f'
```

Comment:
Fails tests. Isn't the correct type (array vs. string) and isn't valid Ruby syntax. Did you forget to run tests locally? `bundle exec rake`